webapp/channels/src/components/start_trial_form_modal/index.tsx

```diff
@@ -64,6 +66,7 @@ function StartTrialFormModal(props: Props): JSX.Element | null {
     const [country, setCountry] = useState('');
     const [businessEmailError, setBusinessEmailError] = useState<CustomMessageInputType | undefined>(undefined);
     const {formatMessage} = useIntl();
+    const canReachCWS = useCWSAvailabilityCheck(true);
```

Comment:
Why is team edition allowed here?

Reply:
Kind of a chicken and egg scenario. Team edition won't have the enterprise code (which contains the availability check) so the availability check will always return a 400. Without this special case, team edition would always be seen as air gapped